### PR TITLE
feat(cli-client): Command-line interface

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,6 @@
     "fix:script": "yarn fix:jsScript && yarn fix:tsScript",
     "fix:style": "prettier --write **/*.scss",
     "fix:tsScript": "prettier --write **/*.ts",
-    "precommit": "lint-staged",
```
Comment (lipis) [inline package.json]: ?

Comment (ffflorian) [reply package.json]: 🤷

Comment (lipis) [reply package.json]: we want that in..

Comment (bennyn) [reply package.json]: Good catch!

**Comment by gregor** [comment]
LGTM

**Pull request merged**